docs/release-notes/NuGet-6.5.md

```diff
@@ -0,0 +1,125 @@
+---
+title: NuGet 6.5 Release Notes
+description: Release notes for NuGet 6.5 including new features, bug fixes, and DCRs.
+author: <TODO: GitHubAlias>
+ms.author: <TODO: MicrosoftAlias>
+ms.date: 2/10/2023
+ms.topic: conceptual
+---
+
+# NuGet 6.5 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.5**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.5](https://visualstudio.microsoft.com/downloads/) | [<TODO: Full SDK Version>](https://dotnet.microsoft.com/download/dotnet-core/<TODO: SDKMajorMinorVersionOnly)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with.NET Core workload
+
+## Summary: What's New in 6.5
+
+* [Feature]: add or update package versions in the Directory.Packages.props file for CPM projects - [#11890](https://github.com/NuGet/Home/issues/11890)
```

Comment:
I'd cleanup the title a bit and remove the `[Feature]`. These don't always have to 100% match the issues, as this is what most of the customers would ready.